src/styles/pages/_visit-requested.scss

```diff
@@ -0,0 +1,138 @@
+@use "../abstracts" as *;
+@use "sass:math";
+
+$page: "p-visit-requested";
+$page-breakpoint: "md";
+$image-size: 16rem;
+
+.#{$page} {
+	display: flex;
+	flex-flow: column nowrap;
+	min-height: 100%;
+	overflow: hidden;
+
+	&__content {
+		flex-grow: 1;
+	}
+
+	&__grid {
+		display: grid;
+		min-height: 100%;
+	}
+
+	&__bottom {
+		position: relative;
+
+		&::after {
+			content: "";
+			display: block;
+			position: absolute;
+			top: 0;
+			right:0;
+			left: 0;
+			bottom: -56px;
```

Comment:
`px` -> `rem`. Ik dacht dat we overal `rem` gingen gebruiken voor consistentie. Correct me if I'm wrong :)